doc/connectivity/networking/api/buf_mgmt.rst

```diff
@@ -1,10 +1,9 @@
 .. _net_buf_mgmt:
 
-Network Buffer Management
+Network Packet Management
```

Comment:
With this rename, the file name seems a bit odd. Should it be renamed to `buf_pkt.rst`? Ditto with the label

Reply:
The label could be changed to `_net_pkt_mgmt`. I am ok with current filename, but if we want to change it, the proper name could perhaps be `pkt_mgmt.rst` or `net_pkt_mgmt.rst`

Reply:
Thank you for reviewing. I have renamed the file and changed the label as suggested above.